src/ui/EastRegion.js

```diff
@@ -542,17 +543,22 @@ EastRegion = function(c) {
                                 }
                             }, {
                                 xtype: 'label',
-                                text: comment.text,
+                                style: 'color: #666; float: right',
+                                hidden: !interpretation.access || !interpretation.access.update,
```

Comment:
This prevents the comment date to be shown, but the comment text is shown regardless of the access rights.

Reply:
That was old code, but it makes no sense as it is. Date and text should be visible even if the user cannot update a comment. So I've removed the hidden tag, does it make sense to you? (I'll update the other PR also, when confirmed)

Reply:
I think it should be fine. The access control should be on the links. If a comment is visible, the whole comment box including the date should be visible.

Reply:
Cool, commit also applied to the v30 branch.